src/Eloquent/Builder.php

```diff
@@ -51,6 +51,13 @@ class Builder extends EloquentBuilder
         'tomql',
     ];
 
+    public function aggregate($function = null, $columns = ['*'])
+    {
+        $result = $this->toBase()->aggregate($function, $columns);
+
+        return $result ?? $this;
```

Comment:
This method must be overrided, because by default this is a fluent interface.

Reply:
We use phpstan on this project, and it get [promoted](https://laravel-news.com/larastan) for Laravel projects.

I have updated the annotation to `@return ($function is null ? AggregationBuilder : self)`, which is the [documented way](https://phpstan.org/writing-php-code/phpdoc-types#conditional-return-types).